kie-spring-boot/kie-spring-boot-samples/kie-server-spring-boot-kafka-sample/src/main/java/org/kie/server/springboot/samples/kafka/WebSecurityConfig.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.server.springboot.samples.kafka;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.crypto.factory.PasswordEncoderFactories;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+@Configuration("kieServerSecurity")
+@EnableWebSecurity
+public class WebSecurityConfig extends WebSecurityConfigurerAdapter {
+
+    @Override
+    protected void configure(HttpSecurity http) throws Exception {
+        http
+        .csrf().disable()
```

Comment:
is this good to show not recommended CSRF disabling? see [sonar complaining about it](https://sonarcloud.io/code?id=org.drools%3Adroolsjbpm-integration&pullRequest=2313&selected=org.drools%3Adroolsjbpm-integration%3Akie-spring-boot%2Fkie-spring-boot-samples%2Fkie-server-spring-boot-kafka-sample%2Fsrc%2Fmain%2Fjava%2Forg%2Fkie%2Fserver%2Fspringboot%2Fsamples%2Fkafka%2FWebSecurityConfig.java&line=36)

Reply:
@mareknovotny I think for this sample it doesn't matter (no production code and tests are not related to auth), I added the annotation _//NOSONAR_ for skipping the analysis in just that line, do you agree?